Dockerfile

```diff
@@ -14,6 +14,10 @@ COPY --from=sbt-cache /root/.cache/coursier ${JENKINS_HOME}/.cache/coursier
 COPY --from=sbt-cache /root/.ivy2 ${JENKINS_HOME}/.ivy2
 COPY --from=sbt-cache /root/.sbt ${JENKINS_HOME}/.sbt
 
+RUN export SDKMAN_DIR="${JENKINS_HOME}/.sdkman" && curl -s "https://get.sdkman.io" | bash
```

Comment:
Maybe I'm being a bit paranoid, but should we lock down a specific version of `sdkman`? I'd hate to having a breaking change come through that could possibly mess up our agents, or is this something we don't have to worry about?